bbb-exporter/collector.py

```diff
@@ -66,6 +81,15 @@ def collect(self):
         bbb_meetings_video_participants.add_metric([], no_video_participants)
         yield bbb_meetings_video_participants
 
+
+        bbb_meetings_participant_clients = GaugeMetricFamily('bbb_meetings_participants_clients',
+                                                            "Total number of participants in all BigBlueButton "
+                                                            "meetings by client",
+                                                            labels=["clientType"])
```

Comment:
I would drop the _client_ prefix of the label like mentioned in the [Prometheus' Best Practices](https://prometheus.io/docs/practices/naming/#labels) documentation. `type="html5|dial-in|flash"` seems reasonable to me.